[issue/19] - Added conclusion content (#27)
teamulster2/report: report/06_conclusion/00_conclusion.md

# Conclusion

> All in all, the developed application with its focus on platform-independence and interchangeability provides a solid basis for further studies on the fatigue assessment topic. Additionally the application based approach brings the advantage of convenient  data collection and representation in order to be analysed.

Reviewer: ```suggestion
All in all, the developed application with its focus on platform-independence
and interchangeability provides a solid basis for further studies on the 
fatigue assessment topic. Additionally the application based approach brings
the advantage of convenient data collection and representation in order to be
analysed.
```

> Basically, all major functions from the original iOS application described in the paper [4] along with the  given user feedback were taken over with well-considered adjustments.

Reviewer: ```suggestion
Basically, all major functions from the original iOS application described in 
the paper [4] along with the  given user feedback were taken over with 
well-considered adjustments.
```

# Conclusion

> All in all, the application and the client-server structure add value 
> to the application based approach to detect cognitive fatigue because 
> it is platform-independent and interchangeable. Basically, all important 
> functions of the previous iOS application designed in the paper [4] were 
> taken over after brief adjustments.

Reviewer 2: ```suggestion
All in all, the developed application with its focus on platform-independence and interchangeability provides a solid basis for further studies on the fatigue assessment topic. Additionally the application based approach brings the advantage of convenient  data collection and representation in order to be analysed.
Basically, all major functions from the original iOS application described in the paper [4] along with the  given user feedback were taken over with well-considered adjustments.
```

In order to generate added value for study purposes, we would recommend 
some further developing, especially in the administration area. 
In addition, analysis tools would have to be created based on our 
export functions, that can provide reliable results for study purposes.

Another suggestion that arose during the design phase was audio or 
video recognition. For this purpose, a model would have to be created 
and trained that recognizes changes in the facial expression or voice 

> of a test person. In interaction with the results from the self-test 

Reviewer 2: ```suggestion
of a test person. In combination with the results from the self-test 
```
